### PR TITLE
plan: Fix a few bugs

### PR DESCRIPTION
**Pull request by zimulala**

### CODE REVIEW

**Comment by shenli** [comment]
@zimulala CI failed.

File: plan/physical_plan_builder.go [hunk 1]
```diff
@@ -40,13 +40,15 @@ func getRowCountByIndexRange(table *statistics.Table, indexRange *IndexRange, in
 		var rowCount int64
 		var err error
 		offset := indexInfo.Columns[i].Offset
-		if l.Kind() == types.KindMinNotNull && r.Kind() == types.KindMaxValue {
+		if l.Kind() == types.KindNull && r.Kind() == types.KindMaxValue {
 			break
+		} else if l.Kind() == types.KindMinNotNull && r.Kind() == types.KindMaxValue {
+			rowCount, err = table.Columns[offset].EqualRowCount(l)
```
Comment (coocood) [inline plan/physical_plan_builder.go]: EqualRowCount(Datum{})

File: plan/physical_plan_builder.go [hunk 2]
```diff
@@ -40,13 +40,15 @@ func getRowCountByIndexRange(table *statistics.Table, indexRange *IndexRange, in
...
+			rowCount = table.Count - rowCount
 		} else if l.Kind() == types.KindMinNotNull {
 			rowCount, err = table.Columns[offset].LessRowCount(r)
```
Comment (coocood) [inline plan/physical_plan_builder.go]: substract equal null.

**Comment by coocood** [comment]
LGTM

**Comment by zimulala** [comment]
@shenli @hanfei19910905 PTAL

**Comment by shenli** [comment]
LGTM

**Pull request merged**